src/ovirtapi/transform.js

```diff
@@ -202,8 +203,15 @@ const VM = {
     }
 
     if (vm.nics && vm.nics.nic) {
+      const reportedDevicesArray =
+        (vm.reported_devices && vm.reported_devices.reported_device)
```

Comment:
With `follow?nics.reporteddevices` reported devices are part of the nic structure. There is no need to handle them externally.